registry/api/v2/errors.go

```diff
@@ -24,7 +24,7 @@ var (
 		Description: `The access controller denied access for the operation on
 		a resource. Often this will be accompanied by a 401 Unauthorized
 		response status.`,
-		HTTPStatusCode: http.StatusForbidden,
+		HTTPStatusCode: http.StatusUnauthorized,
```

Comment:
w/o this change a 403 will be returned instead of a 401. Which would you prefer?